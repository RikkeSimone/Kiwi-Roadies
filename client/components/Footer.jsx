import React from 'react'

function Footer () {
  return (
    <div>
      <div className="footer">
        Created by Kim, Ariana, Rikke and Jonty
      </div>
    </div>
  )
}

export default Footer
