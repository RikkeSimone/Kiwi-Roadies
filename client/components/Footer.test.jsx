import React from 'react'
import { renderWithRedux } from '../testing/utils'
import '@testing-library/jest-dom'
import 'regenerator-runtime/runtime'

import Footer from './Footer'

test('Footer contains names of website creators', async () => {
  const { asFragment } = renderWithRedux(<Footer />)
  expect(asFragment()).toMatchSnapshot()
})
