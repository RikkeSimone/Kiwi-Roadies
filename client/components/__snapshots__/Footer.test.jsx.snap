// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer contains names of website creators 1`] = `
<DocumentFragment>
  <div>
    <div
      class="footer"
    >
      Created by Kim, Ariana, Rikke and Jonty
    </div>
  </div>
</DocumentFragment>
`;
